8Knot/app.py

```diff
@@ -44,8 +44,12 @@
 except SQLAlchemyError:
     sys.exit(1)
 
+# STARTUP OPTIMIZATION: multiselect_startup() disabled for significantly faster startup
```

Comment:
Curious to hear @MoralCode thoughts on this, but I think it would be sufficient to add this in the PR description to refer back to but to remove from codebase. Without the context of the past, seems to be logical that at start up the only repo(s) loaded are the ones listed in the start up value